build.gradle

```diff
@@ -46,12 +35,14 @@ testingConventions.enabled = true
 
 buildscript {
     ext {
-        opensearch_version = "1.3.2"
+        opensearch_version = "2.0.0"
+        bc_version = "1.70"
     }
 
     repositories {
         mavenLocal()
-        maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
+        maven { url "https://plugins.gradle.org/m2/" }
```

Comment:
Why do you need these ones?

Reply:
gradlePluginPortal() should be enough

Reply:
well it is part of `https://github.com/opensearch-project/opensearch-plugin-template-java` template